src/cryptography/hazmat/backends/openssl/decode_asn1.py

```diff
@@ -548,17 +556,17 @@ def _decode_crl_distribution_points(backend, cdps):
             else:
                 rns = cdp.distpoint.name.relativename
                 rnum = backend._lib.sk_X509_NAME_ENTRY_num(rns)
-                attributes = []
+                attributes = set()
```

Comment:
Could you clarify what's going on here? As I understand it this change will make every name entry be added to a single set, which means that all CRLDistributionPoint relativename names are multi-valued?